brianlong/www.validators.app
# 183607579 - move distance and skipped components to the homepage

#### What's this PR do?
Move distance and skipped partials to the homepage and convert them into vue components.

#### How should this be manually tested?
* run tests
* verify if skipped and distance data on the homepage is valid

Remember to check whether the ClusterStat page does NOT contain the above data.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183607579)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
